tests/StackExchange.Redis.Tests/Sets.cs

```diff
@@ -38,6 +38,31 @@ public void SScan()
             }
         }
 
+        [Fact]
+        public void SetContains()
+        {
+            using (var conn = Create())
```

Comment:
New `using` style on tests please! I'll add this to `.editorconfig` as a stronger suggestion so it's automatic.

Reply:
These also need some v6_2_0 checking love!

Reply:
Is that the new style that you meant?

Reply:
If it's right I can fix the style in the other tests in this file :)

Reply:
fixed :)